hi-ml-multimodal/src/health_multimodal/__init__.py

```diff
@@ -2,21 +2,3 @@
 #  Copyright (c) Microsoft Corporation. All rights reserved.
 #  Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
 #  -------------------------------------------------------------------------------------------
-
-BIOMED_VLP_CXR_BERT_SPECIALIZED = "microsoft/BiomedVLP-CXR-BERT-specialized"
```

Comment:
We're moving functions/constants around a lot, and these are part of our public API. Are the changes here important enough to give up on backwards compatibility, why are we choosing to go this route?

We should also be more cautious in deciding what we make part of the public API in the future. https://peps.python.org/pep-0008/#public-and-internal-interfaces

Reply:
I think this variable doesn't need to be public. I agree we could be more cautious about this. I don't have much experience with making these differences, happy to hear suggestions.

As I explained in https://github.com/microsoft/hi-ml/pull/543#discussion_r934416369 and the PR description, some refactoring was needed so the PyTorch Hub method works.

Reply:
Thinking a bit more about this. I think to understand your comment, we'd need to define "our public API". Some of the variables that are being moved are "public" in the sense that they're used by other modules within the library. That doesn't mean we're expecting potential users to use them, I think. The package is not even released, and it doesn't even follow semantic versioning (see #545). Before version 1.0.0, the API should be considered unstable anyway. In summary, maybe we don't need to worry too much about these issues, at least for now.

Reply:
Sounds good to me for this PR. We should think about the public API soon, so we can keep it in mind as we add to the package.